GPU/Common/GPUDefGPUParameters.h

```diff
@@ -61,8 +61,8 @@
 #endif
 
 #define GPUCA_THREAD_COUNT_TRD 512
-#define GPUCA_THREAD_COUNT_CLUSTERER 64 // TODO: why does the result change, if I modify this?
-#define GPUCA_THREAD_COUNT_SCAN 64
+#define GPUCA_THREAD_COUNT_CLUSTERER 256
+#define GPUCA_THREAD_COUNT_SCAN  GPUCA_THREAD_COUNT_CLUSTERER
```

Comment:
I would not link these two numbers.
In principle, there is no reason, why these two steps should run with the same number of threads.
Or do I miss something?
I would tune these individually.
We should also check if the optimal number of threads is the same for all GPUCF kernels.